cloudflare/deploy.ts

```diff
@@ -0,0 +1,66 @@
+import { CF_ACCOUNT_ID, CF_API_TOKEN } from "./env.ts";
+
+const DENOFLARE_VERSION_TAG = "v0.5.12";
+const DENOFLARE_MODULE_URL =
+  `https://deno.land/x/denoflare@${DENOFLARE_VERSION_TAG}`;
+const DENOFLARE_CONFIG_FILENAME = ".denoflare";
+
+if (import.meta.main) {
+  await deploy();
+}
+
+/**
+ * deploy deploys the cloudflare worker to cloudflare.
+ *
+ * https://deno.land/manual/advanced/deploying_deno/cloudflare_workers
+ */
+async function deploy() {
+  // Create a temporary config file for denoflare CLI.
+  const config = {
+    $schema: `${DENOFLARE_MODULE_URL}/common/config.schema.json`,
+    scripts: {
+      /*
+       * TODO: Rename "main" to "lc-dailies" <https://denoflare.dev/guides/serve>.
+       */
+      main: {
+        path: "cloudflare/main.ts",
+        localPort: 8080,
+      },
+    },
+    profiles: {
+      myprofile: {
+        accountId: CF_ACCOUNT_ID,
+        apiToken: CF_API_TOKEN,
+      },
+    },
+  };
+  await Deno.writeTextFile(
+    DENOFLARE_CONFIG_FILENAME,
+    JSON.stringify(config),
+  );
+  console.log({ config });
```

Comment:
TODO: Remove irrelevant console logs.

Reply:
Resolved in <https://github.com/acmcsufoss/lc-dailies/pull/15/commits/d0678303b18a37b52f435ee3a3b5b63a327ee681#diff-854d7a8cbbd35474842f6c0c52644fd53faef44b2baccd0988b78a3b2c16a4ecL41>.